arangod/Aql/QueryCursor.cpp

```diff
@@ -451,6 +460,13 @@ ExecutionState QueryStreamCursor::writeResult(VPackBuilder& builder) {
   if (hasMore) {
     builder.add("id", VPackValue(std::to_string(id())));
   }
+  if (isRetriable()) {
```

Comment:
It looks like this part is identical between QueryResultCursor and QueryStreamCursor. Can it be moved to the shared base class?